backend/btrixcloud/pages.py

```diff
@@ -625,39 +649,35 @@ async def list_pages(
 
             aggregate.extend([{"$sort": {sort_by: sort_direction}}])
 
-        if qa_run_id:
-            aggregate.extend([{"$set": {"qa": f"$qa.{qa_run_id}"}}])
-            # aggregate.extend([{"$project": {"qa": f"$qa.{qa_run_id}"}}])
+        # default sort with search
+        elif search:
+            if is_text_search:
+                aggregate.extend(
+                    [
+                        {"$sort": {"score": {"$meta": "textScore"}}},
+                    ]
+                )
+            else:
+                aggregate.extend([{"$sort": {"url": 1}}])
+        else:
+            # default sort: seeds first, then by timestamp
+            aggregate.extend([{"$sort": {"isSeed": -1, "ts": 1}}])
```

Comment:
This looks like it needs some rationalization with the sort options in the lines above. Ideally we want one $sort stage with primary and any secondary or tertiary sorts included rather than two back-to-back conflicting sort stages.

Reply:
My mistake, I missed the elif in the diff